axelrod/random_.py

```diff
@@ -20,16 +26,14 @@ def random_choice(p=0.5):
     -------
     axelrod.Actions.C or axelrod.Actions.D
     """
-    if p == 0:
-        return Actions.D
 
-    if p == 1:
-        return Actions.C
+    if p in [0, 1]:
+        choice = choices[p]
```

Comment:
It could, but I think multiple exits in a function are best avoided unless there's a good reason for them. Again, more personal preference!